src/usr/local/pkg/miniupnpd.xml

```diff
@@ -209,7 +218,7 @@
 			<fieldname>upload</fieldname>
 			<description>
 				<![CDATA[
-				Value to report when clients query the maximum link upload speed (Kbit/s).
+				Value to report when clients query the maximum link upload speed (kbit/s).
```

Comment:
The current text is fine since the configuration is generated by converting the GUI-provided Kb/s value into kb/s.

Reply:
You're right.
> Bitrates reported by daemon in bits per second

https://github.com/miniupnp/miniupnp/blob/master/miniupnpd/miniupnpd.conf#L130C3-L130C49